src/signals/incident/definitions/wizard-step-2-vulaan/straatverlichting-klokken.ts

```diff
@@ -108,6 +108,18 @@ const straatverlichtingKlokken = {
           typeField: 'objecttype',
           typeValue: '4',
         },
+        {
+          description: 'Het lichtpunt staat niet op de kaart',
+          label: 'Onbekend',
+          icon: {
+            iconUrl: '/assets/images/featureUnknownMarker.svg',
+          },
+          idField: 'id',
+          typeField: 'type',
+          typeValue: UNKNOWN_TYPE,
+        },
```

Comment:
I don't know. This type wasn't added, just moved it.  As for the oak trees, many new trees are planted, especially since 500 trees were lost as a result of the storms.